### PR TITLE
Discharge verification ui design changes.

### PR DESCRIPTION
**Pull request by doug-lovett**
Signed-off-by: Doug Lovett <doug@diamante.ca>

*Issue #:* /bcgov/entity#9083

*Description of changes:*
- General verification report content change to include current/consolidated view information at the time of the registration.
- Update verification general collateral content to include all changes at the time of the registration.
- Discharge verification updates from UI design
- Fix API response for the current view general collateral when adding/deleting in the same registration: include deleted collateral. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-api/src/ppr_api/models/financing_statement.py [hunk 1]
```diff
@@ -209,9 +209,18 @@ def party_json(self, party_type, registration_id):
             if party.party_type == party_type or \
                (party_type == Party.PartyTypes.DEBTOR_COMPANY.value and
                 party.party_type == Party.PartyTypes.DEBTOR_INDIVIDUAL.value):
-                if not self.current_view_json and party.registration_id == registration_id:
+                if party.registration_id == registration_id and \
```
Comment (jordiwes) [inline ppr-api/src/ppr_api/models/financing_statement.py]: just wondering if you can add a few comments on this if block, I'm having trouble following it ;)

**Review by jordiwes — APPROVED** [review]
Just added a few comments, but otherwise looks good

**Review by kialj876 — APPROVED** [review]

File: ppr-api/src/ppr_api/models/financing_statement.py (hunk 1 above)
Comment (doug-lovett) [reply ppr-api/src/ppr_api/models/financing_statement.py]: Comments added.

Comment (jordiwes) [reply ppr-api/src/ppr_api/models/financing_statement.py]: thanks!

**Pull request merged**